CMakeLists.txt

```
@@ -62,6 +62,17 @@ option(BUILD_BENCHMARKS "Enable cpp benchmarks" OFF)
 # Backend
 set(PL_BACKEND "lightning_qubit" CACHE STRING "PennyLane Lightning backend")
 
+# Python bindings are not supported with building multiple backend devices
+list(LENGTH PL_BACKEND PL_BACKEND_LEN)
+if ((${PL_BACKEND_LEN} GREATER 1) AND ENABLE_PYTHON)
+    message(FATAL_ERROR "You cannot build ${PL_BACKEND} simulators with ENABLE_PYTHON")
```

```suggestion
    message(FATAL_ERROR "Lightning does not provide Python support for a multiple backends build.")
```

---

Let's be clear about the value of `PL_BACKEND` here. What do you think of this suggestion?
```suggestion
    message(FATAL_ERROR "Lightning does not provide Python support for multiple backends build: ${PL_BACKEND}")
```

---

I agree. I think you should write like this, though.
```suggestion
    message(FATAL_ERROR "Lightning does not provide Python support for a multiple backends build. Required backends: ${PL_BACKEND}")
```

---

Sorry. `Requested backends:`